api/src/api/licenseStatusRouter.test.ts

```diff
@@ -52,37 +52,18 @@ describe("licenseStatusRouter", () => {
     });
   });
 
-  it("returns user data with updated license status when search is not initiated from a license task", async () => {
-    const licenseData = generateLicenseData({});
-    const userData = generateUserDataForBusiness(generateBusiness({ licenseData }));
-    const nameAndAddress = generateLicenseSearchNameAndAddress({});
-    stubUpdateLicenseStatus.mockResolvedValue(userData);
-
-    // Note: When search is not initiated from a license task, licenseTaskID should be undefined
-    const licenseTaskID = undefined;
-    stubUserDataClient.get.mockResolvedValue(userData);
-    const response = await request(app).post(`/license-status`).send({ nameAndAddress, licenseTaskID });
-    expect(response.status).toEqual(StatusCodes.OK);
-    expect(stubUserDataClient.get).toHaveBeenCalledWith("some-id");
-    expect(stubUserDataClient.put).toHaveBeenCalledWith(userData);
-    expect(response.body).toEqual(userData);
-    expect(stubUpdateLicenseStatus).toHaveBeenCalledWith(userData, nameAndAddress, licenseTaskID);
-  });
-
-  it("returns user data with updated license status when search initiated by user from a license task", async () => {
+  it("returns user data with empty license data when search initiated by user and no license found from a license task", async () => {
```

Comment:
[pebble/dust] pardon my confusion, where in this test are we asserting that licenseData is an empty object?